Makefile

```diff
@@ -25,6 +26,10 @@ publish:
 deploy:
 	aws lambda update-function-code --function-name $(AWS_LAMBDA_PAWS_FUNCTION_NAME) --zip-file fileb://$(AWS_LAMBDA_PAWS_PACKAGE_NAME)
 
+upload:
+	aws s3 cp ./$(AWS_LAMBDA_PACKAGE_NAME) s3://$(AWS_LAMBDA_S3_BUCKET)/packages/lambda/
```

Comment:
Why would we need to upload a generic paws collector package?

Reply:
right, we don't need to upload it.
But we still need to build it just incase someone runs it locally.